ci/build_container/build_recipes/boringssl_fips.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+set -e
+
+# BoringSSL build as described in the Security Policy for BoringCrypto module (2018-10-25):
+# https://csrc.nist.gov/CSRC/media/projects/cryptographic-module-validation-program/documents/security-policies/140sp3318.pdf
+
+# This works only on Linux-x86_64.
+if [[ `uname` != "Linux" || `uname -m` != "x86_64" ]]; then
+  echo "ERROR: BoringSSL FIPS is currently supported only on Linux-x86_64."
+  exit 1
+fi
+
+# BoringSSL commit and checksum from the Security Policy:
+COMMIT=66005f41fbc3529ffe8d007708756720529da20d # fips-20180730
+SHA256=b12ad676ee533824f698741bd127f6fbc82c46344398a6d78d25e62c6c418c73
+
+curl -sLO https://commondatastorage.googleapis.com/chromium-boringssl-docs/fips/boringssl-"$COMMIT".tar.xz \
+  && echo "$SHA256" boringssl-"$COMMIT".tar.xz | sha256sum --check
+tar xf boringssl-"$COMMIT".tar.xz
```

Comment:
delete the tar after extracting? do we want to leave the toolchains in the build image? Feels it is making the build image fat needlessly though I guess we don't really care.